index.bs

```diff
@@ -4064,6 +4061,9 @@ Publishing Registries</h4>
 	is purely documentational,
 	is not subject to the W3C Patent Policy,
 	and <em class=rfc2119>must not</em> contain any requirements on implementations.
+	For the purposes of the Patent Policy [[PATENT-POLICY]],
+	any [=registry section=] in a [=Recommendation track=] document
+	is not a normative portion of that specification.
```

Comment:
Would be good to clarify if this applies to the registry definition or only the registry entries. Seems to me that the registry definition is essentially a data model + a process, and potentially that _could_ somehow be subject to the patent policy, at least theoretically?

Reply:
If that's possible, then we'll need to do it on the Recommendation Track. The Patent Policy only applies to the Rec Track...

Reply:
This is where life gets complicated. The registry definition part of the registry section is (though I doubt it matters); the registry tables are not. (But that should be clear from the fact that the section contains no normative requirements and there are no expressed normative requirements about its content).

Reply:
We probably need to get PSIG review on whether registry definitions should or should not be subject to the Patent Policy, but in any case, if they need to be then they need to be on the REC track, and if they do not need to be, then it's better for them to not be on the REC track. The Patent Policy only applies to documents on the REC track.

Reply:
https://www.w3.org/Consortium/Patent-Policy-20200915/#sec-Licensing

Reply:
That said, I think we should be clear about what that would do: if the definitions of the registry get to be on the REC track and to be covered by the Patent Policy, what would be protected is implementing the definitions: in other words, *creating and operating the registry tables*. This would do nothing for software implementations of whatever is described in the registry tables. Are we actually concerned that we'd design a registry so that editing it and publishing it could be a violation of someone's patents, and that someone could attempt to sue the W3C for our editorial practices for that registry? That seems far-fetched to me.

Reply:
> Are we actually concerned that we'd design a registry so that editing it and publishing it could be a violation of someone's patents, and that someone could attempt to sue the W3C for our editorial practices for that registry? That seems far-fetched to me.

Me too, and yet, I'm willing to bet that someone somewhere has tried such a trick, or may do in the future (call me a cynic!). Maybe we could sidestep this and avoid potential unnecessary complexity if someone is able to document publicly (maybe in the guide, or some other technical report) typical and useful existing registry patterns that are known to be patent-free.

Reply:
Maybe I'm just underinformed, but I am not aware of any standards organization ever being sued because the way they develop documents infringes on someone's patent, nor of any standards organization having some form of patent protection against that. Unlike patent protection for implementations of specifications, which is very valuable, this seems to me to be a theoretical exercise of very limited interest.